### PR TITLE
[New Rule] Suspicious managed code hosting process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/203

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_suspicious_managedcode_host_process.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/21"
+ecs_version = ["1.5.0"]
```
Comment (brokensound77) [inline rules/windows/defense_evasion_suspicious_managedcode_host_process.toml:3]:
```suggestion
ecs_version = ["1.6.0"]
```

File: rules/windows/defense_evasion_suspicious_managedcode_host_process.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious Managed Code Hosting Process which could indicate code injection or other form of suspicious code
+execution. Verify process details such as commandline, parent process and network connections.
```
Comment (brokensound77) [inline rules/windows/defense_evasion_suspicious_managedcode_host_process.toml:11]:
```suggestion
execution. Verify process details such as command line, parent process, and network connections.
```

File: rules/windows/defense_evasion_suspicious_managedcode_host_process.toml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious Managed Code Hosting Process which could indicate code injection or other form of suspicious code
+execution. Verify process details such as command line, parent process, and network connections.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious managed code hosting process"
```
Comment (brokensound77) [inline rules/windows/defense_evasion_suspicious_managedcode_host_process.toml:16]:
```suggestion
name = "Suspicious Managed Code Hosting Process"
```

File: rules/windows/defense_evasion_suspicious_managedcode_host_process.toml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/08/21"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious Managed Code Hosting Process which could indicate code injection or other form of suspicious code
```
Comment (brokensound77) [inline rules/windows/defense_evasion_suspicious_managedcode_host_process.toml:10]:
```suggestion
Identifies Suspicious Managed Code Hosting process which could indicate code injection or other form of suspicious code
```

**Review by brokensound77 — APPROVED** [review]
just the 2 comments, then LGTM

File: rules/windows/defense_evasion_suspicious_managedcode_host_process.toml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+creation_date = "2020/08/21"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious Managed Code Hosting process which could indicate code injection or other form of suspicious code
+execution. Verify process details such as command line, parent process, and network connections.
```
Comment (bm11100) [inline rules/windows/defense_evasion_suspicious_managedcode_host_process.toml:11]:
```suggestion
Identifies a suspicious managed code hosting process which could indicate code injection or other form of suspicious code
execution. 
```
Would move to `note` - 
> Verify process details such as command line, parent process, and network connections.

**Review by bm11100 — APPROVED** [review]
one suggestion on description, otherwise lgtm.

**Pull request merged**